bindings/storage.i

```diff
@@ -95,6 +95,7 @@
 %include "../../storage/Utils/HumanString.h"
 %include "../../storage/Utils/Region.h"
 %include "../../storage/Utils/Remote.h"
+%include "../../storage/Utils/Logger.h"
```

Comment:
Wrong ordering can cause problem for swig. Here I was just grouping subdirectories.